src/pages/Credentials/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+
+import { createToken, deleteToken, listTokens } from '@/services/credentialsService';
+import './styles.scss';
+
+type CredentialsProps = {
+  workspaceProject: string;
+};
+
+export const Credentials = ({ workspaceProject }: CredentialsProps) => {
+  const [tokens, setTokens] = useState([]);
+
+  useEffect(() => {
+    fetchTokens();
+  }, []);
+
+  const fetchTokens = async () => {
+    try {
+      const fetchedTokens = await listTokens();
+      setTokens(fetchedTokens);
+    } catch (error) {
+      console.error('Failed to fetch tokens:', error);
+    }
+  };
```

Comment:
I'm surprosed eslint didn't flag this as being used before it is defined. I would probably have this function inside the useEffect, it is after all the only place it is used.

Reply:
Ah very true! Think it has something to do with hoisting. 

I've moved it into the useEffect for now.